example/README.md

```diff
@@ -0,0 +1,2 @@
+example
```

Comment:
Extend this description a little bit, at least to mention that this project boots up an http server and explain that after you boot it up, you should be able to open localhost:[whatever] and see a beautiful webpage